controllers/user.controller.js

```diff
@@ -0,0 +1,60 @@
+const Users = require('../models/user.model');
+
+// get all users
+exports.getAllUsers = async (req, res, next) => {
+    try {
+        const users = await Users.find();
+        res.status(200).json(users);
+    } catch (err) {
+        res.status(500).json({message: err.message});
```

Comment:
So firstly..There is a utility file in the utility folder called response.utility. This helps with sending json responses from the api.